css/settings.scss

```diff
@@ -12,8 +12,10 @@ $color-1: #2C2F33;
 $color-2: #3B3F42;
 $color-3: #D8D8D8;
 $color-4: rgba(#2C2F32, .1);
+$color-4-1: #2C2F32;
```

Comment:
There's no difference between this and `color-1`, please remove this one and use it.